CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ the code was deployed.
 
```

Comment:
Can you please expand on your Test Plan in the PR description?

I would expect to see you try deploying it to dev and making sure that both English and Bilingual French/English text messages work when they include the `ALERT_TYPE.BUTTONS_URGENT` value.